content/docs/welcome-guide/create/creating-projects-using-project-manager.md

```diff
@@ -63,24 +67,26 @@ If this is the first time you've used Project Manager to create a project, then
 
     ![Or choose "New Project - Create New Project"](/images/welcome-guide/project-manager-menu-create-new-project.png)
```

Comment:
This comment is for step 2 under **Configure the engine settings**, where it says "If you setup and built O3DE from GitHub..."

Change "setup" to "set up" since you are using it as a verb.